### ISSUE TITLE
Top-level calls to permissions.navigator.query for SAA should probably return "granted"

### ISSUE THREAD

**Issue by johannhof** [privacycg/storage-access#168]
In Chrome they currently return "prompt", which isn't accurate as the spec explicitly mandates no prompt or heuristics be applied here.

**Comment by johannhof** [comment]
Update: This was fixed a while back in Chromium by https://chromium-review.googlesource.com/c/chromium/src/+/4418027

Probably needs a spec change in https://privacycg.github.io/storage-access/#permissions-integration though.

**Comment by annevk** [comment]
Hmm, but wouldn't that imply that if you delegate it there is no prompt?

**Comment by johannhof** [comment]
Delegate in what sense? Via permissions policy?

**Comment by annevk** [comment]
Yeah. I guess this permission is double-keyed so it's a bit different. Prolly bound to be confusion either way.
